docs/meetings/2023/TechArea20230718.md

```diff
@@ -38,21 +38,28 @@ Release
 -   AI (Mat): Release XRootD 5.6+
 -   AI (BrianL): Release xrootd-multiuser-2.1.3+
 -   AI (Matt): osg-ca-scripts missing dependency on perl-filetest.
-    We need this done before we can build EL9 tarballs
--   AI (Matt): Release oidc-agent-4.4.2+
+    We need this done before we can build EL9 tarballs - no longer included with base perl packages in EL9
```

Comment:
```suggestion
    We need this done before we can build EL9 tarballs - no longer included with base perl packages in EL9.
    MatS to provide special Perl packaging tips.
```

Friendly reminder @matyasselmeci

Reply:
derp I had the comment written and forgot to hit 'save'. Fixed.